rust/rope/src/engine.rs

```diff
@@ -110,7 +119,7 @@ impl Engine {
 
     /// Get text of head revision.
     pub fn get_head(&self) -> Rope {
-        self.rev_content_for_index(self.revs.len() - 1)
+        self.text.clone()
```

Comment:
Yes, it's cheap.

At the back of my head I'm probably thinking about the impact on the "mutate in place" optimization - if a clone of the text is retained it could defeat that optimization, while returning &Rope would force the user to drop the reference before doing any mutating change. But I'm not sure that's an issue in practice.